03_Uitvoering.Rmd

```diff
@@ -2668,16 +2561,158 @@ Elke gridcel waar het plaatsen van een pb gewenst is, wordt eerst gecategoriseer
+tubes_selected <- bind_rows(tubes_cat1 %>% mutate(cat = "1"), 
+                            tubes_cat2_bis %>% mutate(cat = "2"),
+                            tubes_cat3_grts %>% mutate(cat = "3"))
+
+tubes_selected <- tubes_selected %>% 
+  select(-selectie) %>% 
+  left_join(tubes_eval_namenyanthes %>% 
+              dplyr::select(loc_code, selectie), by = "loc_code")
+
+tubes_reserve <- bind_rows(tubes_cat3_grts_reserve %>% mutate(cat = "3")
+)
+tubes_reserve <- tubes_reserve %>% 
+  select(-selectie) %>% 
+  left_join(tubes_eval_namenyanthes %>% 
+              dplyr::select(loc_code, selectie), by = "loc_code")      
+
+output_vc <- write_vc(tubes_selected, file.path(".","data","tubes_selected"), sorting = c("loc_code"), strict =  FALSE)
+output_vc <- write_vc(tubes_reserve, file.path(".","data","tubes_reserve"), sorting = c("loc_code"), strict =  FALSE)
+rm(output_vc)
+```
+
+
+Tabel \@ref(tab:synthese-selection-tubes) toont het overzicht van alle `r nrow(tubes_selected)` geselecteerde peilbuizen.
+Hiervan zijn er `r nrow(tubes_selected %>% filter(selectie ==1))` die op basis van hun data nu reeds inzetbaar zijn.
+
+
+Tabel \@ref(tab:synthese-reserve-tubes) geeft alle `r nrow(tubes_reserve)` reservepunten weer.
+
+
+```{r synthese-selection-tubes}
+
+kobj <- kable(tubes_selected %>%
+      mutate('direct inzetbaar' = if_else(selectie == 1, 'Ja', 'Nee')) %>% 
+      dplyr::select(rasterid, groupnr, area_name, loc_code, x, y, 
+             'direct inzetbaar') %>%
+      rename(GTgroep = groupnr,
+             rastercelnummer = rasterid,
+             gebied = area_name,
+             watinacode =  loc_code
+             ) %>%
+      arrange(rastercelnummer, GTgroep),
+      caption =  "Overzicht van de geselecteerde peilbuizen"
+      )
+if (pdfmaken == TRUE) {
+  kobj
+} else {
+  kobj %>%
+    kable_styling(bootstrap_options = c("striped", "hover", "condensed", "responsive"),
+                full_width = T,
+                position = "left",
+                font_size = 8,
+                fixed_thead = T
+                ) %>%
+   # column_spec(1:3, bold = F, border_right = F, width = "35em") %>%
+   # column_spec(c(3,5:7, 9:10), width = "1cm") %>%
+   # row_spec(0, angle = -90, align = "c")  %>% #fixeer veldnamen
+    scroll_box(height = "250px")
+}
+
+```
+
+<br/><br/>
+
+```{r synthese-reserve-tubes}
+
+kobj <- kable(tubes_reserve %>%
+        dplyr::select(rasterid, groupnr, area_name, loc_code, x, y) %>%
+        rename(GTgroep = groupnr,
+               rastercelnummer = rasterid,
+               gebied = area_name,
+               watinacode =  loc_code
+               ) %>%
+        arrange(rastercelnummer, GTgroep),
+      caption =  "Overzicht van de reserve peilbuizen"
+      )
+if (pdfmaken == TRUE) {
+  kobj
+} else {
+  kobj %>%
+    kable_styling(bootstrap_options = c("striped", "hover", "condensed", "responsive"),
+                full_width = T,
+                position = "left",
+                font_size = 8,
```

Comment:
I guess font size will need to be larger for scroll boxes (see general comments of my previous review). I didn't recompile to check.

Reply:
changed it to 12. It seems ok so.